lte/gateway/deploy/agw_prepare.sh

```diff
@@ -38,6 +38,6 @@ fi
 mkdir -p /home/magma/.ssh
 chown magma:magma /home/magma/.ssh
 # Removing incompatible Kernel version
-apt remove -y linux-image-4.9.0-11-amd64
+DEBIAN_FRONTEND=noninteractive apt remove -y linux-image-4.9.0-11-amd64
```

Comment:
This seems legit but I don't currently have access to hardware to verify, and any breakages to this script can be painful to debug after provisioning a box. How did you test this change?

Reply:
I've just tested this single step with apt, but I can make a full deploy on Monday to make sure everything goes smoothly.

Reply:
Finally I've made a full install with this patch applied. The result, as expected, was:

```
magma@magma:~$ uname -a
Linux magma 4.9.0-9-amd64 #1 SMP Debian 4.9.168-1 (2019-04-12) x86_64 GNU/Linux
```